### PR TITLE
Switch to Graphiti gem for JSON API generation.

### PR DESCRIPTION
**Pull request by plural**
This is more actively maintained and a bit simpler than JSONAPI::Resources which we had been using.

The central problem to solve is to allow cross-linked relationships between public and private objects, which JSONAPI::Resources did not allow. The resource classes are a bit tidier as well and we have a bit more control without a lot of effort.

Apologies in advance for this being A Big Pull Request.  This also clears out the old files and regenerates all the docs.

### CODE REVIEW

File: config/master.key.old [hunk 1]
```diff
@@ -0,0 +1 @@
+e6611b28126c3e7b98ec79b7a8afc3ee
```
Comment (NoahTheDuke) [inline config/master.key.old:1]: intentional?

Comment (plural) [reply config/master.key.old]: lol no

Thanks for driving by, Noah.  :D

Comment (NoahTheDuke) [reply config/master.key.old]: 👁️ 👄 👁️

**Comment by plural** [comment]
I'm moving this into main to unblock a bunch of other stuff.

**Pull request merged**